src/dodal/beamlines/i03.py

```diff
@@ -94,19 +95,6 @@ def vfm_mirror_voltages(
     )
 
 
-@skip_device(lambda: BL == "s03")
-def hfm(
```

Comment:
How come this device is removed now?

Reply:
So initially when I first started working on the energy changes, I based it off of what GDA currently does, however after Neil put me right it turns out that we don't actually do any changes with it, everything is handled with feedback so no need to move it. Only the vertical mirror is needed.